support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -277,6 +277,7 @@ export function ThreeTierLanding({
 	const urlSearchParams = new URLSearchParams(window.location.search);
 	const urlSearchParamsProduct = urlSearchParams.get('product');
 	const urlSearchParamsRatePlan = urlSearchParams.get('ratePlan');
+	const urlSearchParamsOneTime = !urlSearchParams.get('oneTime');
```

Comment:
i.e. true if the parameter exists

Reply:
Although I've just had a play and this doesn't quite work because if the querystring is just `?oneTime` then it's also false because the value is empty.
How about this instead:
`urlSearchParams.has('oneTime');`

Reply:
Strangely if I do that the urlSearchParamsOneTime is coming up as false and so the url is not working as expected

Reply:
> Although I've just had a play and this doesn't quite work because if the querystring is just `?oneTime` then it's also false because the value is empty.
> How about this instead:
> `urlSearchParams.has('oneTime');`

So yes, this totally makes sense, thank you!